tests/util/keyring.py

```diff
@@ -195,7 +195,6 @@ def cleanup(self):
         if self.delete_on_cleanup:
             self.keychain.keyring_wrapper.keyring.cleanup_keyring_file_watcher()
             temp_dir = self.keychain._temp_dir
-            print(f"Cleaning up temp keychain in dir: {temp_dir}")
             shutil.rmtree(temp_dir)
```

Comment:
One could in that case also just pass the member instead of the introduced handle `temp_dir`, which is now not used beyond the line that follows.

Reply:
Updated